Sources/TokamakDOM/DOMRenderer.swift

```diff
@@ -18,43 +18,19 @@
 import JavaScriptKit
 import TokamakCore
 
-public final class DOMNode: Target {
-  let ref: JSObjectRef
-  private var listeners: [String: JSClosure]
-
-  init<V: View>(_ view: V, _ ref: JSObjectRef, _ listeners: [String: Listener] = [:]) {
-    self.ref = ref
-    self.listeners = [:]
-    super.init(view)
-    reinstall(listeners)
-  }
-
-  init<A: App>(_ app: A, _ ref: JSObjectRef, _ listeners: [String: Listener] = [:]) {
-    self.ref = ref
-    self.listeners = [:]
-    super.init(app)
-    reinstall(listeners)
-  }
-
-  /// Removes all existing event listeners on this DOM node and install new ones from
-  /// the `listeners` argument
-  func reinstall(_ listeners: [String: Listener]) {
-    for (event, jsClosure) in self.listeners {
-      _ = ref.removeEventListener!(event, jsClosure)
-    }
-    self.listeners = [:]
+extension EnvironmentValues {
+  /// Returns default settings for the DOM environment
+  static var defaultEnvironment: Self {
+    var environment = EnvironmentValues()
+    environment[_ToggleStyleKey] = _AnyToggleStyle(DefaultToggleStyle())
+    environment[keyPath: \._defaultAppStorage] = LocalStorage.standard
+    _DefaultSceneStorageProvider.default = SessionStorage.standard
 
-    for (event, listener) in listeners {
-      let jsClosure = JSClosure {
-        listener($0[0].object!)
-        return .undefined
-      }
-      _ = ref.addEventListener!(event, jsClosure)
-      self.listeners[event] = jsClosure
-    }
+    return environment
   }
 }
 
+// FIXME: doesn't this belong to TokamakCore?
```

Comment:
`SpacerContainer` is part of TokamakDOM, as not all renderers will handle flexible sizing the way browsers do. Their parent element could already know that if a child is requesting full width, then it needs to expand.

Although, it also makes sense to move this to TokamakCore so that any Renderer that does require this information doesn't need to do so from scratch...